azure-pipelines.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 ##############################
 - job: Check_Build_Options
   pool:
-    vmImage: 'windows-latest'
+    vmImage: 'windows-2022'
```

Comment:
We're good with using `windows-latest`. 
At the time of this writing, the information I have is that, going forward, this will have the expected Windows+VS combination. If we stick with `windows-latest` we won't have to change the pipeline yaml in the future, unless we want it "locked" into a specific version.

Reply:
Done.